dear_petition/petition/export/forms.py

```diff
@@ -138,6 +140,22 @@ def map_offenses(self):
             self.data[f"DOOF:{i}"] = self.format_date(ciprs_record.offense_date)
             self.data[f"DismissalDate:{i}"] = self.format_date(offense.disposed_on)
 
+    def map_same_day_offenses(self):
+        dismissed_records = self.petition.get_all_offense_records()
+        charges = dismissed.same_day_convictions(dismissed_records)
+        line = []
+        for charge in charges[:1]:
```

Comment:
Since the logic for both the first charge and the rest are almost identical maybe we can do this instead:

```python
        for index, charge in enumerate(charges):
            formatted = f"{charge.offense.ciprs_record.file_no} {charge.description.title()}"
            if index == 0:
                self.data["ChargedDesc"] = formatted
            else:
                self.data["ChargedDescCont"].append(f" {formatted}")
```